ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -134,6 +135,57 @@ if tflite_support_is_available
   )
 endif
 
+if tflite2_support_is_available
+  filter_sub_tflite2_sources = ['tensor_filter_tensorflow_lite.cc']
+
+  nnstreamer_filter_tflite2_sources = []
+  foreach s : filter_sub_tflite2_sources
+    nnstreamer_filter_tflite2_sources += join_paths(meson.current_source_dir(), s)
+  endforeach
+
+  nnstreamer_filter_tflite2_deps = tflite_support_deps + [thread_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep]
+
+  if tflite_nnapi_delegation_is_available
```

Comment:
This is unnecessary (nnfw delegation in tf-lite subplugin)
I made a patch to remove this feature, plz delete this line.

Reply:
Applied. Thanks.